solana/pyth2wormhole/program/src/types/mod.rs

```diff
@@ -51,12 +53,16 @@ pub const PUBKEY_LEN: usize = 32;
 #[repr(u8)]
 pub enum PayloadId {
     PriceAttestation = 1,
```

Comment:
Is this to maintain the ability to attest single prices, or could the batch be used with an `N` of 1 for the same purpose? Curious mostly to confirm if this is going to be the standard format for P2W messages in case it needs to be documented.

Reply:
My instinct was to leave it in place so that there can be no confusion about what is being sent